session1/doc/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
```

Comment:
There are too many files committed in PR. Please make sure you are committing only relevant files. Folders like node_modules should be git ignored